gulpfile.js

```diff
@@ -4,7 +4,7 @@ const gulp = require('gulp')
 const parallel = require('async/parallel')
 const series = require('async/series')
 const createTempRepo = require('./test/utils/create-repo-nodejs.js')
-const HTTPAPI = require('./src/http-api')
+const HTTPAPI = require('./src/http')
```

Comment:
It makes sense to call this `http` module now `daemon`

Reply:
@diasdavid Yeah it does, but in `go-ipfs` it's `http` , so should we stick to the reference implementation ? I don't mind the rename though

Reply:
Let's keep http then :)